scripts/LSSTComCam/runOneOffCalexp.py

```diff
@@ -0,0 +1,67 @@
+# This file is part of rubintv_production.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import lsst.daf.butler as dafButler
+from lsst.rubintv.production.oneOffProcessing import OneOffProcessor
+from lsst.rubintv.production.podDefinition import PodDetails, PodFlavor
+from lsst.rubintv.production.utils import getAutomaticLocationConfig, getDoRaise
+from lsst.summit.utils.utils import setupLogging
+
+setupLogging()
+instrument = "LSSTComCam"
+
+workerNum = 0
+
+locationConfig = getAutomaticLocationConfig()
+# detectorNumber = None becuase this is a per-instrument type pod
+# detector number below is the actual CCD which will be processed, which
+# should not be controlled by k8s or the head node, and therefore goes there
```

Comment:
I don't understand this comment (maybe it's missing a negation?).  There isn't a detector number below, and I don't know what "there" is.

Reply:
Wow, re-reading that now, even _I_ don't understand past-Merlin, that comment is garbage. Rewriting 🙏